[build]: sequelize model 추가

thdwlsgus0 opened the pull request:
1.  milestone과 issue_image를 관리하기 위해서 sequelize model을 추가하였습니다.
2. index.js는 충돌이 날까봐 push하지 않았습니다.

* 의논할 사항
   - local db를 만들지 여부
   - 만들면 production과 development의 구분이 필요할 것 같습니다.
   - [블로그](https://velog.io/@qksud14/oneroomtell-04)
   
* 향 후 진행할 사항
   - react공부 - zerocho 강의 2강
   - sequelize 공부 
   - wiki 수정
   - github oauth 공부 및 적용

thdwlsgus0 committed:
[build]: sequelize model 추가

1.  milestone과 issue_image를 관리하기 위해서 sequelize model을 추가하였습니다.

mu1616 commented on server/models/milestone.js:
```diff
@@ -0,0 +1,36 @@
+               type: DataTypes.STRING(1500),
+               allowNull: false,
+           },
+           due_date:{
+               type:'TIMESTAMP',
+               defaultValue: sequelize.literal('current_timestamp'),
+               allowNull:false,
+           },
```
마일스톤의 due_date는 해당 내용이 입력되지 않아도 마일스톤이 생성이 가능하다고 기획서 24번슬라이드에 나와있습니다. 그렇기 때문에 defaultValue, allowNull 부분은 빼도 될 것 같네요!

thdwlsgus0 replied:
수정 하겠습니다!! 감사합니다!!

mu1616 commented on server/models/issue_image.js:
```diff
@@ -0,0 +1,26 @@
+            primaryKey: true,
+            autoIncrement: true,
+         },
+           issue_id:{
+             type: DataTypes.INTEGER,
+             allowNull: false,
```
issue_id는 belongsTo를 통해 관계 설정을 하면 자동으로 추가되기 때문에 따로 속성으로 추가하지 않아도 될 것 같습니다.

thdwlsgus0 replied:
수정하겠습니다!!

mu1616 reviewed (commented):
수고많으셨습니다!! 리뷰를 한번 달아봤는데 확인한번만 해주세용 :smile:

thdwlsgus0 committed:
[refactor] : 코드 리팩토링

1. sequelize의 객체 부분의 네이밍을 camelCase로 적용하였습니다.
2. model의 이름을 '-'컨벤션을 이용해서 적용하였습니다.

thdwlsgus0 committed:
[refactor] : 코드 리팩토링

1. milestone의 due_date 속성 중 defaultValue와 allowNull 속성을 제거하였습니다.
2. issue_id가 관계 설정을 하면 자동 생성되기 때문에 삭제해주었습니다.
